controllers/halayerset_controller_test.go

```diff
@@ -175,52 +178,104 @@ var _ = Describe("High Availability Layer Set CR", func() {
 })
 
 func verifyCmdCommands(action userAction) {
+	time.Sleep(time.Second * 2)
```

Comment:
Why do you add a delay, sleep of two seconds?

Reply:
Good point, added a comment